app/code/Magento/Catalog/Model/Product/Option/Type/Text.php

```diff
@@ -58,7 +58,7 @@ public function validateUserValue($values)
         parent::validateUserValue($values);
 
         $option = $this->getOption();
-        $value = trim($this->getUserValue());
+        $value = ($this->getUserValue() !== null) ? trim($this->getUserValue()) : '';
```

Comment:
Hello @SilinMykola 
Please remove redundant brackets

```suggestion
        $value = $this->getUserValue() !== null ? trim($this->getUserValue()) : '';
```

Reply:
Hello @andrewbess 
Thanks for your review. I made changes, please check.